include/screenshot.h

```diff
@@ -0,0 +1,15 @@
+#ifndef SCREENSHOT_H
+#define SCREENSHOT_H
+
+
+struct xdpw_ppm_pixel {
+	unsigned char red, green, blue;
+};
+
+struct xdpw_ppm_image {
+	unsigned int width, height;
+	int max_color_value;
+	struct xdpw_ppm_pixel *data;
```

Comment:
In practice we'll only need a single pixel, maybe we can simplify all of this by just having `red, green, blue` directly embedded here?

Reply:
Sure, but then we don't really need width/height either, especially if we exit out if those are not 1?

Reply:
Yup! We can get rid of those as well.

I don't think there'll be a situation where we'll want to parse more than one pixel worth of PPM data.